cloud/logger/src/Logger.ts

```diff
@@ -0,0 +1,8 @@
+interface Logger {
+  debug(message: string, data?: any, source?: string, tags?: string[]): void;
```

Comment:
the source is the file in which it logs - it's very helpful in browserified apps.
will add generic log